Resources/Prototypes/_NF/Entities/Clothing/Back/base_clothing_backpack.yml

```diff
@@ -0,0 +1,13 @@
+- type: entity
+  parent: [ ClothingBackpack,] # DeltaV - Comment out NFBackpackHiddenStash
+  id: NFClothingBackpack
+  components:
+  - type: Storage
+    maxItemSize: Large
+    defaultStorageOrientation: Vertical
+    grid:
+    - 0,0,6,3
+  - type: Item
+    size: Huge
+    shape:
+    - 0,0,6,3
```

Comment:
This file and proto isn't needed, its just a normal backpack at that point, remove this file